packages/badge/src/react/index.tsx

```diff
@@ -1,13 +1,15 @@
-import { css } from 'glamor'
-import React from 'react'
-import PropTypes from 'prop-types'
-
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
```

Comment:
i've been removing filterReactProps hoping that with static types we no longer need to be defensive....i think that is safe for most needs and definitely should be ok for this simple component

Reply:
Yes, great catch. Thanks!